docs/source/developer/contributing.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to HIVE
+
+This simple guide will walk you through the process of contributing to HIVE.
+
+## Table of Contents
+
+- Identifying an Issue
+- Letting Others Know You're Working on the Issue
+- Forking the Repository
+- Opening a Pull Request
+
+## Identifying an Issue
+
+1. Go to the "Issues" tab in the repository.
+1. Look for issues with labels such as "good first issue" if you're new to the project.
+1. Read the issue description and any associated comments to make sure you understand the problem and its requirements.
+1. Choose an issue that aligns with your skills and interests.
+
+## Letting Others Know You're Working on the Issue
+
+1. Leave a comment on the issue, stating that you'd like to work on it. This helps prevent duplication of effort and lets us know someone is addressing the problem.
+1. Wait for a response from the maintainers or the issue creator. They may provide additional information or guidance, or assign the issue to you.
```

Comment:
we can't assign the issues to people who fork it, can we?

Reply:
Looks like if they comment on the issue we can assign it to them: https://github.blog/changelog/2019-06-25-assign-issues-to-issue-commenters/